File: javascript/길이에 따른 연산.js
const solution = (num_list) => {
    let answer = 0;
    for(let i of num_list) {
        if(num_list.length > 10) {
            answer += i;
        }else {
            if(answer == 0){
                answer = 1;
            }
            answer *= i;
        }
    }
    return answer;
}

File: java/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        for(int i = 0; i < num_list.length; i++) {
            if(num_list.length > 10) {
                answer += num_list[i];
            }else {
                if(answer == 0){
                    answer = 1;
                }
                answer *= num_list[i];
            }
        }
        return answer;
    }
}